lib/omnibus-ctl.rb

```diff
@@ -694,6 +694,11 @@ def run(args)
 
       command_to_run = args[0]
 
+      if command_to_run == "--help"
+        help
+        Kernel.exit 0
+      end
```

Comment:
I can add a comment, if that would help. Adding `--help` to the command map didn't seem to work because there is some transformation happening along the way to turn --help into __help.

Another option would be to strip out the `--` and just make it `help.` Would that be less confusing?
```ruby
 if command_to_run == "--help"
   command_to_run = "help"
 end
```